source/System.cpp

```diff
@@ -99,7 +100,7 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 
 	// For the following keys, if this data node defines a new value for that
 	// key, the old values should be cleared (unless using the "add" keyword).
-	set<string> shouldOverwrite = {"asteroids", "attributes", "belt", "fleet", "link", "object", "hazard"};
+	set<string> shouldOverwrite = {"asteroids", "attributes", "belt", "fleet", "hazard", "link", "object"};
```

Comment:
why are we changing the order here?

Reply:
Alphabetical
Warp asked earlier when I was changing this and I think its fine to keep it?

Reply:
yea fine, was just confused